packages/react/package.json

```diff
@@ -23,12 +23,12 @@
     "url": "https://github.com/vmware/clarity.git"
   },
   "dependencies": {
-    "@lit-labs/react": "^1.0.1",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1"
+    "@lit-labs/react": "^1.0.1"
   },
   "peerDependencies": {
-    "@cds/core": "./dist/core"
+    "@cds/core": "./dist/core",
+    "react":"^16.13.1 || ^17.0.2",
```

Comment:
Do we want to relax this further like `>16.13.1`?

Reply:
Probably - that would include the React 18 beta, which you may or may not want to support yet